src/components/pages/Offers/Offer/Stocks/ActivationCodesUploadDialog/__specs__/UploadedFileChecker.spec.js

```diff
@@ -71,7 +71,7 @@ describe('uploadedFileChecker', () => {
 
       // then
       expect(errorMessage).toStrictEqual(
```

Comment:
```suggestion
      expect(errorMessage).toBe(
```
`toStrictEqual` n'est uniquement que pour les objets.